pavelib/utils/test/suites/bokchoy_suite.py

```diff
@@ -23,7 +25,9 @@ class BokChoyTestSuite(TestSuite):
       log_dir - directory for test output
       report_dir - directory for reports (e.g., coverage) related to test execution
       xunit_report - directory for xunit-style output (xml)
-      fasttest - when set, skip various set-up tasks (e.g., DB migrations)
+      fasttest - when set, skip various set-up tasks (e.g., collectstatic)
```

Comment:
Just collectstatic. (And compiling static assets, if I remember right.) DB migrations definitely DO run when using fasttest. That, in part, is because we blow away the database on each (traditional) run.